data/json/items/gun/22.json

```diff
@@ -134,7 +131,6 @@
     "recoil": 5,
     "durability": 9,
     "burst": 20,
-    "reload": 500,
```

Comment:
This gun is specifically supposed to have a very high capacity, but hard to reload magazine, what's the rationale for reversing that?